metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -174,56 +189,62 @@ def delete(
             "This will permanently delete data from DataHub. Do you want to continue?",
             abort=True,
         )
-
-    if urn:
-        # Single urn based delete
+    if urn or csv:
+        if csv:
+            df = pd.read_csv(csv, header=None, names=["urn"])
```

Comment:
This isn't really a csv - it's just a list of urns in a file. Might make sense to change the naming a bit, maybe "urns from file" or similar

Also I'd like to avoid taking a dependency on pandas in the core cli - so it'd be better to use the built in python file.read().split('\n') to get the urns

Reply:
Ok i refactored it, thanks!

Reply:
@hsheth2 do you mind helping to review again? thanks!